Makefile

```diff
@@ -5,12 +5,43 @@ MESHERY_CLOUD_DEV=http://localhost:9876
 MESHERY_CLOUD_PROD=https://meshery.layer5.io
 MESHERY_CLOUD_STAGING=https://staging-meshery.layer5.io
 
+
```

Comment:
@Aisuko you may want to replicate a version of these changes to https://github.com/layer5io/meshery/blob/master/mesheryctl/Makefile

Reply:
Done